### PR TITLE
Expose storage altering functions in EVM Executor Backend

### PR DESCRIPTION
**Pull request by ArielTM**
## Motivation

Working with Foundry EVM Executor, one can alter balance, nonce and code of an account, but not the storage.

## Solution

This PR exposes two functions
1. `active_fork_db_mut` will be `pub` (same as `active_fork_mut`)
2. Added the function `insert_account_storage` to the Backend

### CODE REVIEW

File: evm/src/executor/backend/mod.rs [hunk 1]
```diff
@@ -467,6 +467,23 @@ impl Backend {
         }
     }
 
+    pub fn insert_account_storage(
```
Comment (mattsse) [inline evm/src/executor/backend/mod.rs:470]: needs one-liner doc

**Review by mattsse — CHANGES_REQUESTED** [review]
reasonable,

doc nit

**Review by Evalir — APPROVED** [review]
thank you! should be useful.

**Review by mattsse — APPROVED** [review]

**Comment by ArielTM** [comment]
Is there anything needed to merge this?

**Comment by mattsse** [comment]
nope, just forgot -.-

**Pull request merged**